main.py

```diff
@@ -1,7 +1,27 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name = "Western",)
+    Genre.objects.create(name="Action",)
+    Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in line 17, but it's better to use the correct name from the start.